app/src/main/java/eu/kanade/tachiyomi/data/preference/PreferenceKeys.kt

```diff
@@ -9,6 +9,7 @@ object PreferenceKeys {
     const val lightTheme = "light_theme"
     const val darkTheme = "dark_theme"
     const val themeDarkAmoled = "pref_theme_dark_amoled_key"
+    const val themeContrastAmoled = "contrast_amoled_key"
```

Comment:
Yeah don't make this a preference, I'd rather you just updated amoled and see if people complain

Reply:
I think still be useful to have a choice

Reply:
I'm not suggesting, I'm saying don't make it a preference. This is just bloat as a preference

Reply:
> I'd rather you just updated amoled and see if people complain

First need to wait for the pull to be accepted